### ISSUE TITLE
ComboBox Cannot Navigate Dropdown When Narrator is ON

### ISSUE THREAD

**Issue by genevasun** [OfficeDev/office-ui-fabric-react#10887]
### Environment Information

- **Package version(s)**: 6.207.0
- **Browser and OS versions**: Microsoft Edge 44.18362.387.0 Windows 10

### Describe the issue:

With Edge+Narrator, I cannot access the ComboBox dropdown with down-arrow key. 
Also reproduced on https://developer.microsoft.com/en-us/fabric#/controls/web/combobox.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/genevasun/pen/oNNzoxw

#### Actual behavior:

Select the ComboBox. Use Enter key to open drop down. Use down-arrow key to focus on the first item in the drop down. Nothing happens.

#### Expected behavior:

The first item in the drop down should be focused, and I can navigate the drop down with up and down arrow keys.

**Comment by marygans** [comment]
Hi @genevasun, thank you for submitting this issue!
I was able to test the issue using Edge + Narrator and I am able to repro in scan mode and wondering if you are experiencing this in scan mode as well? 

If so, there was an [issue](https://github.com/OfficeDev/office-ui-fabric-react/issues/10689) recently closed related to this that I found related to this and was recently closed due to it potentially being an Narrator issue. 
Below is an example without scan mode:
![combobox-4](https://user-images.githubusercontent.com/38991459/67052184-3d238280-f0f2-11e9-8bc7-c86ac60061b0.gif)

**Comment by genevasun** [comment]
@marygans The issue you referred to seems to be the same issue. Can I have more information on the Narrator bug so I can track its progress? I did not find that information in the other issue.

**Comment by marygans** [comment]
https://microsoft.visualstudio.com/OS/_workitems/recentlyupdated/

> @marygans The issue you referred to seems to be the same issue. Can I have more information on the Narrator bug so I can track its progress? I did not find that information in the other issue.

@genevasun if you are a Microsoft employee, I can send you a link to their internal bug tracking repo offline. Feel free to reach out - my alias is @marygans.

**Comment by genevasun** [comment]
Closing because this is a Narrator issue.
